DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ScheduleCell.java

```diff
@@ -101,8 +112,20 @@ public void updateItem(Session item, boolean empty) {
         }
     }
 
-    private void updateVBox() {
+    private void updateListTile() {
         if (session.getTalk() != null) {
+            if (session.getTalk().getTrack() != null) {
+                final String trackId = session.getTalk().getTrackId().toUpperCase();
+                
+                getStyleClass().removeAll(styleClasses);
+                getStyleClass().add(fetchStyleClassForTrack(trackId));
+                
+                final Label track = new Label(trackId);
```

Comment:
Avoid creating a Label/Group/StackPane every time, add them on the constructor and just update the label's text